eos/form-factors/parametric-bgl1997-impl.hh

```diff
@@ -197,27 +211,48 @@ namespace eos
     }
 
     double
-    BGL1997FormFactors<BToDstar>::t_1(const double & /*s*/) const
+    BGL1997FormFactors<BToDstar>::t_1(const double & s) const
     {
-        return 0.0;  //  TODO
+        // resonances for 1^-, which have overlap with the tensor current
+        const double blaschke = _z(s, 6.329 * 6.329) * _z(s, 6.910 * 6.910) * _z(s, 7.020 * 7.020);
+        const double phi      = _phi(s, _t_0, 24, 3, 3, 2, _chi_T_1p);
+        const double z        = _z(s, _t_0);
+        const double series   = _a_T1[0] + _a_T1[1] * z + _a_T1[2] * z * z + _a_T1[3] * z * z * z;
```

Comment:
`const double series = _a_T1[0] + z * (_a_T1[1] + z * (_a_T1[2] + z * _a_T1[3]));` is far more efficient
Idem below

Reply:
I will put it on the post-merge cleanup pile.

Reply:
Understood